COMPATIBILITY.md

```diff
@@ -88,6 +88,7 @@ Entries list the class name serving as the entry point to the relevant functiona
 - [X] `bitbucket`: supported as of 1.1.2
 - [ ] `bitbucket-pullrequest-builder`: [JENKINS-31749](https://issues.jenkins-ci.org/browse/JENKINS-31749)
 - [ ] `xtrigger`: [JENKINS-31933](https://issues.jenkins-ci.org/browse/JENKINS-31933)
+- [ ] `stashNotifier`: [issue 92](https://github.com/jenkinsci/stashnotifier-plugin/issues/92)
```

Comment:
Actually `StashNotifier` is a post-build action, not a trigger, so this belongs in a different section.